CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ a release.
 ---
 
 ## [Unreleased]
+### Added
+- SoftDeleteable: Add option to enable or disable handling of the `postFlush` event (#2958)
```

Comment:
IIUC, this PR should also have at least an entry for the `Changed` and `Fixed` sections, because the added option changes the way the previous fix behaves, while fix the issues mentioned in the description.

Could you please confirm?

Reply:
Hi. Yes you're right. I've updated the changelog.